iwyu_output.cc

```diff
@@ -1800,6 +1801,9 @@ LineSortOrdinal GetLineSortOrdinal(const OneIncludeOrForwardDeclareLine& line,
       return LineSortOrdinal::AssociatedInlineDefinitions;
     return LineSortOrdinal::AssociatedHeader;
   }
+
+  if (GlobalFlags().quoted_headers_first && EndsWith(quotedInclude, "\""))
```

Comment:
Oops, I just noticed now that `quotedInclude` doesn't follow the `hacker_case` naming convention. I'll push a commit to fix that, you may need to resolve conflicts when rebasing.

Reply:
I can include in my fix if you prefer.